here-naksha-lib-handlers/src/main/java/com/here/naksha/lib/handlers/SourceIdHandler.java

```diff
@@ -92,11 +85,11 @@ protected EventProcessingStrategy processingStrategyFor(IEvent event) {
 
   private void transformPropertyOperation(ReadFeatures readRequest) {
 
-    if (readRequest.getPropertyOp() == null) {
+    if (readRequest.getQuery().getProperties() == null) {
```

Comment:
Is `readRequest.getQuery()` null-safe?

Reply:
yes, `NotNullProperty<ReadRequest, RequestQuery>(RequestQuery::class)`